src/main/java/co/omise/models/Charge.java

```diff
@@ -91,7 +91,9 @@ public class Charge extends Model {
     @JsonProperty("captured_amount")
     private long capturedAmount;
     @JsonProperty("webhook_endpoints")
-    private List<String> webhookEndpoints; 
+    private List<String> webhookEndpoints;
+    @JsonProperty("merchant_advice")
+    private String merchantAdvice;
```

Comment:
What about `failure_code` and `failure_message`? did you add them as well?
<img width="782" alt="image" src="https://github.com/omise/omise-java/assets/122003737/07c62a8d-6cfb-49d0-bb4d-7916f29e4e76">
<img width="332" alt="image" src="https://github.com/omise/omise-java/assets/122003737/f62353b8-e576-44a8-975d-94fb2ac04350">

Reply:
Actually I already see those fields in the Charge Model
![Uploading image.png…]()